### ISSUE TITLE
Feature Request: UI option to open native IDE

### ISSUE THREAD

**Issue by samkearney** [microsoft/vscode-cmake-tools#1274]
### Brief Issue Summary

**TL;DR:** Would you consider adding a button/menu option to open native IDE projects generated by CMake?

My development workflow has me switching back and forth between VSCode and Visual Studio to develop CMake C++ projects on Windows (I prefer using native Visual Studio for debugging). The CMake tools extension handles every part of my CMake workflow except one: opening the IDE project after generation, e.g. with the command:

```
cmake --open [build directory]
```

This is the one thing I still need to switch to either a terminal or explorer window for. Would you consider adding a UI feature for this? I could envision it as part of the three-dot menu on the extension sidebar.

**NOTE:** If this feature already exists and I've missed it, please feel free to tell me so and close. Thanks!!

**Comment by bobbrow** [comment]
I think you can write a shell task in `tasks.json` that does this.  Use `${command:cmake.buildDirectory}` for `[build directory]`

https://code.visualstudio.com/Docs/editor/tasks#_custom-tasks

**Comment by samkearney** [comment]
Nice, I hadn't thought of that, thanks! I still think it might be nice to have it be part of the extension, but this works pretty well for me as a workaround.

**Comment by gcampbell-msft** [comment]
Closing this due to the age of the issue, lack of upvotes, and we likely won't get to this issue. Thanks!
